packages/visualizations-react/package.json

```diff
@@ -105,16 +111,16 @@
         "jest-environment-jsdom": "^29.0.2",
         "jest-fetch-mock": "^3.0.3",
         "postcss": "^8.4.16",
-        "react": "^16.13.0",
-        "react-dom": "^16.13.0",
-        "react-is": "^16.13.0",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
         "rollup": "^4.18.0",
         "rollup-plugin-postcss": "^4.0.2",
         "size-limit": "^8.1.0",
         "storybook": "^7.6.17",
         "storybook-addon-rtl": "^1.0.0",
         "tslib": "^2.1.0",
-        "typescript": "^5.5.3"
+        "typescript": "^5.5.3",
+        "vite": "^5.4.14"
```

Comment:
👀 Leftover from a test ? 😄

Reply:
No, vite is used by storybook now and I don't know why in the previous iteration it found its own vite but I had to install explicitely this time 🤔